psi4/src/psi4/libmints/overlap.cc

```diff
@@ -336,7 +336,7 @@ void OverlapInt::compute_pair_deriv2(const GaussianShell& s1, const GaussianShel
                             buffer_[(0*size)+ao12] += (4.0*a1*a1*x[l1+2][l2]*y[m1][m2]*z[n1][n2] -
                                                       2.0*a1*(2*l1+1)*x[l1][l2]*y[m1][m2]*z[n1][n2]) * over_pf;
                             if (l1 > 1)
```

Comment:
Hmm, I think you just never tested anything beyond P orbitals. This `if` statement will not be hit without D or higher.

I wish we could put weights on commits so that we could crank this one up. Three lines changed, but I am sure quite a bit of time looking into it.

Reply:
Three _characters_ changed, but painful ones to track down, I'm sure.

I'll apply these changes to pyvib2 and see if tests/python/vibanalysis/ magically heals.

A thousand thanks for investigating this.